vars/sectionDeployToAKS.groovy

```diff
@@ -35,49 +36,52 @@ def call(params) {
 
   if (config.dockerBuild) {
 
-    withAcrClient(subscription) {
-      acr = new Acr(this, subscription, env.REGISTRY_NAME, env.REGISTRY_RESOURCE_GROUP)
+    withAcrClient(registrySubscription) {
+      acr = new Acr(this, registrySubscription, env.REGISTRY_NAME, env.REGISTRY_RESOURCE_GROUP)
       dockerImage = new DockerImage(product, component, acr, new ProjectBranch(env.BRANCH_NAME).imageTag(), env.GIT_COMMIT)
       onPR {
         acr.retagForStage(DockerImage.DeploymentStage.PR, dockerImage)
       }
     }
-    withAksClient(subscription, environment) {
+    withAksClient(registrySubscription, environment) {
```

Comment:
Do we need this ? I mean changing to registrySubscription when withSubscription is the next effective line and nothing in between ? Not sure if i am missing something.